packages/react-native/src/types/Module.ts

```diff
@@ -426,7 +426,8 @@ export interface Module {
    * Use the other `IOSNotificationSettings` properties to view which specific permissions were
    * allowed.
    *
-   * On Android, all of the properties on the `IOSNotificationSettings` interface response return
+   * On Android, `alert` will return `0` if permission is denied, `1` if permission is given.
+   * The rest of the properties on the `IOSNotificationSettings` interface response return
    * as `AUTHORIZED`.
    *
    * @platform ios
```

Comment:
no longer just ios - same for one below it on line 461/463

Reply:
I assume I will have to delete the `@platform ios` line, correct?

Reply:
exactly that! nice